.versionbot/CHANGELOG.yml

```diff
@@ -184,7 +184,7 @@
   title: "'patch: updated electron to 12.2.3'"
   date: 2022-02-21T08:33:45.382Z
 - commits:
-    - subject: "patch: fix mesage of null"
+    - subject: "patch: fix message of null"
```

Comment:
Some repositories don't like changing changelogs. I'm happy to drop these changes.

Reply:
This is an auto-generated file from commit messages, should not be edited manually.